client/js/lounge.js

```diff
@@ -193,7 +195,9 @@ $(function() {
 		);
 		renderChannel(data.chan);
 		var chan = sidebar.find(".chan")
-			.sort(function(a, b) { return $(a).data("id") - $(b).data("id"); })
+			.sort(function(a, b) {
+				return $(a).data("id") - $(b).data("id");
+			})
```

Comment:
I'm not against having this one a single line to be honest, but if that helps with other inconsistencies, sure, why not.
If we start having ES6-style arrow functions, I think having this on a single line will be beneficial however:

```js
.sort((a, b) => $(a).data("id") - $(b).data("id"))
```